templates/company/company_name_availability/form.html.tx

```diff
@@ -14,14 +14,8 @@
             <form id="name-availability-search" action="/company-name-availability" method="get" accept-charset="utf-8" role="search" class="search-header js-search-hash">
 
                 <div class="searchfield">
-                    <input type="search"
-                           autocomplete="off"
-                           name="q"
-                           id="company-name-availability-search-text"
-                           title="Search"
-                           value="<% $query %>" placeholder="Search"
-                           class="govuk-input"/>
-                    <button type="submit" id="search-submit" class="search-submit gem-c-search__submit">Search</button>
+                    <input type="search" class="govuk-input" autocomplete="off" name="q" id="company-name-availability-search-text" title="Search" value="<% $query %>" placeholder="Search"/>
+                    <button type="submit" id="search-submit" class="search-submit govuk-button">Search</button>
```

Comment:
This appears to be undoing the changes made for https://companieshouse.atlassian.net/browse/BI-11373?